docs/pages/debugging.md

```diff
@@ -0,0 +1,26 @@
+Debugging ArrayFire Issues {#debugging}
+===============================================================================
+
+Some general advice for debugging mysterious issues with ArrayFire follows.
+
+First, try running with additional debugging information:
+
+    export AF_PRINT_ERRORS=1           #Make exceptions more helpful
+    export AF_TRACE=all                #Print all the stuff
+    #export AF_TRACE=mem               #Print memory management allocation, free, and garbage collection information
+    #export AF_TRACE=unified           #Print unified backend dynamic loading information
+    #export AF_TRACE=mem,unified       #Print both of the above
+    export AF_JIT_KERNEL_TRACE=kernels #Write out the source code generated by the JIT to the directory `kernels`
+    ./my_program                       #Run program with the above
+
+Next, add some debugging output to your code.
```

Comment:
Another section should start here - "Tips for debugging in Language Bindings" or something like that.

Reply:
Fixed.